app/models/host/managed_extensions.rb

```diff
@@ -34,6 +34,10 @@ def setReboot
     true
   end
 
+  def delReboot
```

Comment:
> not related tot his patch?

It is, it removes exception from logs about missing rollback method when
transaction fails.

-- 
Later,
 Lukas #lzap Zapletal